src/workerd/api/queue.h

```diff
@@ -98,7 +98,8 @@ struct IncomingQueueMessage {
   kj::Date timestamp;
   kj::Array<kj::byte> body;
   kj::Maybe<kj::String> contentType;
-  JSG_STRUCT(id, timestamp, body, contentType);
+  uint16_t attempts;
```

Comment:
I assume this will be documented in the docs as well, but does this start at 0 for the first attempt or at 1? It'd be worth noting here so that it's in the code as well as in the docs.

Perhaps the ambiguity suggests a different name would be better (`previousAttempts`, `pastAttempts`, etc.), but those come with their own downside in terms of wordiness, and they assume that this starts at 0, but judging by your tests I'm guessing this actually starts at 1 on the first attempt.

Reply:
I thought that comments here got automatically picked up and put into the typescript types for public consumption, but now that I look at https://www.npmjs.com/package/@cloudflare/workers-types?activeTab=code I see that most of the types don't have comments on their fields, so I guess it's not a big deal. Plus it'd be nice to merge these before conflicts start happening, so I'll get this in now.